src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx

```diff
@@ -74,6 +78,17 @@ describe('<FilesPage>', () => {
     getByText('Claim status is unavailable');
   });
 
+  // it('should call setPageFocus with lastPage and loading after 100ms when location.hash is empty', done => {
+  //   shallow(<FilesPage {...props} />);
+
+  //   setTimeout(() => {
+  //     expect(setPageFocusSpy.calledOnce).to.be.true;
+  //     // expect(setPageFocusSpy.calledWith(props.lastPage, props.loading)).to.be
+  //     //   .true;
+  //     done();
+  //   }, 110); // Allow a bit more time than the timeout in the component
+  // });
+
```

Comment:
Should this test (and its dependencies) just be deleted instead of commented out?
If not, I think there should be a comment left in as to why it was commented out for the sake of our future selves 😛

Reply:
updated! - rewrote test and uncommented it out!